packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1658,6 +1658,8 @@
   "Report_this_message_question_mark": "Report this message?",
   "Request": "Send Request",
   "Request_to_join_room": "You are not a member of this #<strong>__roomName__</strong> room. Would you like to send request to join",
+  "Request_to_join_room_comment": "A reason, you would like to join this room",
```

Comment:
Proposal: The reason why would you like to join this room

de: Der Grund, warum Du dem Kanal beitreten möchtest

Reply:
translation maintained.